src/features/course-player/CoursePlayer.jsx

```diff
@@ -62,14 +60,14 @@ const CoursePlayer = (props) => {
   }, [courseId, routeUnitId, saveSequencePosition, saveUnitPosition, sequenceId, sequenceStatus, unitIds]);
 
   // checkFetchCourse
-  useEffect(() => {
-    fetchCourse(routeCourseId);
-  }, [fetchCourse, routeCourseId]);
+  // useEffect(() => {
```

Comment:
Why comment this? did you change how this works?

Reply:
I removed that now, yes i changed how this works, now the course and sequences are fetched from CourseView this is because CourseView use this data in many places than CoursePlayer like Sidebar and also I'm not fetching only one sequence anymore I'm fetching all the sequences for the sidebar.